src/posthog-core.ts

```diff
@@ -1036,13 +1038,19 @@ export class PostHog {
             properties
         )
 
-        const property_blacklist = this.config.property_blacklist
-        if (_isArray(property_blacklist)) {
-            _each(property_blacklist, function (blacklisted_prop) {
-                delete properties[blacklisted_prop]
+        // since property_blacklist is deprecated in favor of property_denylist, we merge both of them here
+        const property_denylist = [...this.config.property_blacklist, ...this.config.property_denylist]
```

Comment:
also feels like we should merge this once, i think maybe we have to do it in init and in `set_config`

that way internally the SDK only has a denylist, and existing config gets copied into it whenever we evaluate the config

Reply:
discussed offline, and added a TODO for now, it requires a bit of test refactoring to make it work since the override option always replaces the set_config call.